### PR TITLE
Real Device Cleaning Process  documentation

### PR DESCRIPTION
**Pull request by aoleszyn**
### Description
Real Device Cleaning Process  documentation

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

**Comment by kristofmuhi** [comment]
Approved

File: docs/mobile-apps/real-device-cleaning.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+id: real-device-cleaning
+title: Real Device Cleaning Process
+sidebar_label: Real Device Cleaning Process
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The cleaning process runs at the end of each test session for Sauce Labs Real Device. It resets the device back to a default state, ready for the next test. The cleaning process also removes any private data which may have been part of the prior test. If we encounter any issue during the cleaning process, or we fail to clean the device, we put those devices aside for manual inspection to prevent data and security leaks. 
+
+The cleaning process differs slightly between the Public Devices which are shared among all our customers, and the Private Devices, where the owner has more control over the configuration of their device. It also differs between Android and iOS, due to the different features of these operating systems.
```
Comment (lysannep) [inline docs/mobile-apps/real-device-cleaning.md:10]:
```suggestion
The cleaning process differs slightly between the Public Devices, which are shared among all our customers, and the Private Devices, where the owner has more control over the configuration of their device. It also differs between Android and iOS, due to the different features of these operating systems.
```

File: docs/mobile-apps/real-device-cleaning.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Every 5th cleaning session includes a device reboot. This ensures that the system remains 
+performant, and any stray processes are stopped.
+
+:::note
+A factory reset is not performed between test sessions, as this requires manual intervention and would slow down the time required between test runs.
+:::
+
+### Cleaning Process Steps
+
+We use a proprietary process that wipes every real device clean at the end of the testing session:
```
Comment (lysannep) [inline docs/mobile-apps/real-device-cleaning.md:23]: Do these steps always occur in this order? If so, the list could be reformatted as a numbered list.

File: docs/mobile-apps/real-device-cleaning.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Private Devices follow the same process as Public Devices. However, the customer can adjust the following as desired:
+
+App allowlist:
+* Apps installed on the device can be selected, by name/bundleID/package name, to be preserved between sessions. These apps and their data will not be cleaned. They can be uploaded by the customer from Sauce Storage, or installed from the App Store/Play Store.
+  
+Account allowlist: 
+* Google accounts signed into the Play Store can be preserved between sessions.
+* Apple IDs signed in to the device and App Store can be preserved between sessions.
+  
+## Goals of the Cleaning Process
```
Comment (lysannep) [inline docs/mobile-apps/real-device-cleaning.md:51]: This section could be moved higher up in the page.

**Review by lysannep — APPROVED** [review]
Added a few comments, otherwise looks good to me!

File: docs/mobile-apps/real-device-cleaning.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The cleaning process runs at the end of each test session for Sauce Labs Real Device. It resets the device back to a default state, ready for the next test. The cleaning process also removes any private data which may have been part of the prior test. If we encounter any issue during the cleaning process, or we fail to clean the device, we put those devices aside for manual inspection to prevent data and security leaks. 
+
+The cleaning process differs slightly between the Public Devices which are shared among all our customers, and the Private Devices, where the owner has more control over the configuration of their device. It also differs between Android and iOS, due to the different features of these operating systems.
+
+## Public Devices
+
+Every 5th cleaning session includes a device reboot. This ensures that the system remains
```
Comment (aliciakrebs) [inline docs/mobile-apps/real-device-cleaning.md:14]:
```suggestion
Every fifth cleaning session includes a device reboot. This ensures that the system remains 
```

File: docs/mobile-apps/real-device-cleaning.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+The cleaning process differs slightly between the Public Devices which are shared among all our customers, and the Private Devices, where the owner has more control over the configuration of their device. It also differs between Android and iOS, due to the different features of these operating systems.
+
+## Public Devices
+
+Every 5th cleaning session includes a device reboot. This ensures that the system remains 
+performant, and any stray processes are stopped.
+
+:::note
+A factory reset is not performed between test sessions, as this requires manual intervention and would slow down the time required between test runs.
```
Comment (aliciakrebs) [inline docs/mobile-apps/real-device-cleaning.md:18]: This sentence is still a little hard to understand. I think it was meant to say that a factory reset would make the time between test runs take longer. 
```suggestion
A factory reset is not performed between test sessions, as this requires manual intervention and would increase the time required between tests.
```

File: docs/mobile-apps/real-device-cleaning.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+## Private Devices
+
+Private Devices follow the same process as Public Devices. However, the customer can adjust the following as desired:
+
+App allowlist:
+* Apps installed on the device can be selected, by name/bundleID/package name, to be preserved between sessions. These apps and their data will not be cleaned. They can be uploaded by the customer from Sauce Storage, or installed from the App Store/Play Store.
+  
+Account allowlist: 
+* Google accounts signed into the Play Store can be preserved between sessions.
+* Apple IDs signed in to the device and App Store can be preserved between sessions.
```
Comment (aliciakrebs) [inline docs/mobile-apps/real-device-cleaning.md:49]:
```suggestion
* Apple IDs signed into the device and App Store can be preserved between sessions.
```

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**